src/actions/ProductsActions.ts

```diff
@@ -0,0 +1,63 @@
+import { Notify } from 'quasar';
+import { getListOfProductsQuery } from '../services';
+import { useQuery } from '../utils/apollo';
+
+interface getListOfProductsReturnTypes {
+  products: object[];
+  totalProducts: number;
+}
+
+interface getListOfProductsTypes {
+  limit: number;
+  offset: number;
+  query: string;
+  order_by: getListOfProductsOrderByTypes;
+}
+
+interface getListOfProductsOrderByTypes {
+  order_by_price?: any;
+  order_by_name?: any;
+}
+
+export const getListOfProducts = ({
+  limit = 10,
+  offset = 0,
+  query = '',
+  order_by = {
+    order_by_price: null,
+    order_by_name: null,
+  },
+}: getListOfProductsTypes) => {
+  query = `%${query}%`;
+
+  return new Promise<getListOfProductsReturnTypes>((res, rej) => {
```

Comment:
Hmm maybe leave the full name of "res" and "rej", even if is more verbose

Reply:
Also in other files

Reply:
Thanks! I'll fix it